kinesis_video_streamer/CMakeLists.txt

```diff
@@ -58,11 +58,22 @@ install(FILES kvs_log_configuration DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATI
 
 if(CATKIN_ENABLE_TESTING)
     find_package(rostest REQUIRED)
+    
+    include_directories(/usr/include/gmock /usr/src/gmock)
```

Comment:
The solution used by @AAlon on https://github.com/aws-robotics/cloudwatchmetrics-ros1/commit/ed8a9a92e11b599d53c7bb60bef7004d31601756 to enable gmock looks cleaner

Reply:
Please avoid that solution, that leads to "Unable to find gmock", see https://github.com/aws-robotics/lex-common/pull/1 for context